fmf/_compat_jsonschema.py

```diff
@@ -0,0 +1,91 @@
+"""Compatibility layer for jsonschema validation."""
+
+from __future__ import annotations
+
+import warnings
+from typing import Any, Dict, List, NamedTuple, Optional, Union
```

Comment:
Let's not mix `__future__.annotation` and `typing.Dict` for now. We should take a hammer at it eventually though.